src/features/create-dao/steps/DAOSummary.module.scss

```diff
@@ -0,0 +1,70 @@
+.Row {
+	display: flex;
+	margin-top: 0.625rem;
+	width: 20.5rem;
+
+	&_Summary {
+		display: flex;
+		margin-top: 0;
+	}
+
+	.Column {
+		display: flex;
+		flex-direction: column;
+		
+		.Heading {
+			margin: 0 0 1rem 0;
+		}
+		
+		.MediaInput {
+			height: 12.5rem;
+			width: 12.5rem;
+			margin-right: 2rem;
+			border-radius: 0.5rem !important;
+		}
+		
+		.Summary {
+			margin-right: 1.3rem;
+			width: 8rem;
+			height: 2.5rem;
+			margin-bottom: 1rem;
+		}
+
+		&_Second {
+			line-height: 1.5rem;
+		}
+	}
+}
+
+.Text {
+	font-family: 'Inter';
+	font-style: normal;
+	font-weight: 400;
+	font-size: 1rem;
+	line-height: 1.5rem;
+
+	&_Bold {
+		font-weight: 700;
+	}
+
+	&_Confirmation {
+		color: #F58C61;
+		text-align: center;
+		width: 35rem;
+		height: 4.5rem;
+		border-top: solid 0.063rem #232323;
+		padding: 1.5rem 0;
+		margin-top: 1.25rem;
+		margin-bottom: 1rem;
+	}
+
+	&_Description {
+		margin-top: 1.5rem;
+		margin-bottom: 0;
+		line-height: 1.5rem;
+	}
+}
+
+.Footer {
+	margin-top: 1rem;
```

Comment:
`gap` would be good to use over margins between these sections. Can circle back to this.